frame/staking/src/lib.rs

```diff
@@ -690,34 +718,73 @@ decl_storage! {
 		Nominators get(fn nominators):
 			linked_map hasher(blake2_256) T::AccountId => Option<Nominations<T::AccountId>>;
 
-		/// Nominators for a particular account that is in action right now. You can't iterate
-		/// through validators here, but you can find them in the Session module.
+		/// The current era index.
 		///
-		/// This is keyed by the stash account.
-		pub Stakers get(fn stakers):
-			map hasher(blake2_256) T::AccountId => Exposure<T::AccountId, BalanceOf<T>>;
+		/// This is the latest planned era, depending on how session module queues the validator
+		/// set, it might be active or not.
+		pub CurrentEra get(fn current_era): Option<EraIndex>;
 
-		/// The currently elected validator set keyed by stash account ID.
-		pub CurrentElected get(fn current_elected): Vec<T::AccountId>;
+		/// The active era index, the era currently rewarded.
+		///
+		/// Validator set of this era must be equal to `SessionInterface::validators`.
+		pub ActiveEra get(fn active_era): Option<EraIndex>;
 
-		/// The current era index.
-		pub CurrentEra get(fn current_era) config(): EraIndex;
+		/// The start of the active era.
+		pub ActiveEraStart get(fn active_era_start): Option<MomentOf<T>>;
```

Comment:
These storage items seem to always be accessed together. We should combine them into a single item.

Reply:
not in `fn convert`, `fn on_offence`, `start_session`, `end_session` but I guess grouping them doesn't harm indeed.